spec/services/remark_generators/orchestrator_spec.rb

```diff
@@ -43,13 +59,21 @@ module RemarkGenerators
 
       expect(ResidualBalanceChecker).to receive(:call).with(liquid_capital_items, 0, 100).and_call_original
 
-      described_class.call(liquid_capital_items:,
-                           state_benefits:,
-                           lower_capital_threshold: 100,
-                           child_care_bank: 0,
-                           outgoings: childcare_outgoings + housing_outgoings + legal_aid_outgoings + maintenance_outgoings,
-                           employments:,
-                           gross_income_summary: assessment.applicant_gross_income_summary, assessed_capital: 0)
+      orchestrator
+    end
+
+    context "pre MTR" do
+      it "return RemarksData array without priority_debt remark" do
+        expect(orchestrator).not_to include(RemarksData.new(type: :priority_debt, issue: :priority_debt, ids: []))
```

Comment:
can we not say remarks is empty?

Reply:
@starswan I am checking for an element in a array thats why not using empty keyword